File: Api/Boardgame/Boardgame.Domain/Entities/Notification.cs
using Boardgame.Domain.Entities.Common;

namespace Boardgame.Domain.Entities;

public class Notification
{
    public Guid Id { get; set; }
    public DateTime CreateAt { get; set; }
    public TypeNotification Type { get; set; } = TypeNotification.Success;
    public string TableId { get; set; } = string.Empty;
    public NotificationStatus NotificationStatus { get; set; } = NotificationStatus.Online;
    public Guid UserId { get; set; }
    public UserIdentity? User { get; set; }
}

File: webadmin/src/models/data/notification.ts
export interface SendSuccessRequest {
  userId: string;
  tableId: string;
}

export interface SendNotSuccessRequest {
  userId: string;
}

export interface NotificationResponseHub {
  Id: string;
  CreateAt: string;
  Type: number;
  TableId: string;
  NotificationStatus: number;
  UserId: string;
}
